tests/unit/test__main.py

```diff
@@ -19,7 +19,9 @@
 
 from data_validation import cli_tools, consts
 from data_validation import __main__ as main
+from tests.system.data_sources.test_bigquery import BQ_CONN
 
+PROJECT_ID = os.getenv("PROJECT_ID")
```

Comment:
Where do you use this variable?

Reply:
Only BQ_CONN variable that's been imported needs it, actually